From 6382ca542ec2fd765b65bf6abc52640e8a0d19b3 Mon Sep 17 00:00:00 2001
From: Jon Roberts <jon@emptyflask.net>
Date: Fri, 26 Jun 2020 00:40:38 -0500
Subject: [PATCH 6/6] fira code

---
 config.def.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config.def.h b/config.def.h
index 26861f0..46e854d 100644
--- a/config.def.h
+++ b/config.def.h
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char *font = "Fira Code:pixelsize=14:antialias=true:autohint=true";
 static int borderpx = 2;
 
 /*
-- 
2.25.4

